src/pudl/metadata/resources/eia.py

```diff
@@ -4,6 +4,18 @@
 from pudl.metadata.codes import CODE_METADATA
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "balancing_authorities_eia": {
+        "description": "A coding table describing balancing authorities in EIA-860 and EIA-923.",
+        "schema": {
+            "fields": ["code", "label", "description"],
+            "primary_key": ["code"],
+            "foreign_key_rules": {"fields": [["balancing_authority_code_eia"]]},
+        },
+        "encoder": CODE_METADATA["balancing_authorities_eia"],
+        "sources": ["eia860"],
+        "etl_group": "static_eia",
```

Comment:
And this is now automatically loaded into the DB because all of the `static_eia` tables are grabbed in the `etl` module?

Reply:
yup!